test/float_tex.cpp

```diff
@@ -79,7 +80,9 @@ static void updateFloatTexture() {
     glBindTexture(GL_TEXTURE_2D, 0);
     alpha -= 0.001;
 }
+
 static void glut_draw_callback(void) {
+    printf("glut_draw_callback\n");
```

Comment:
do we want to keep these printfs in the test?

Reply:
Sure, I think it better overall to have more print statements in the test.